internal/runtime/runtime.go

```diff
@@ -0,0 +1,91 @@
+package runtime
+
+import (
+	"awans.org/aft/internal/datatypes"
+	"fmt"
+	"github.com/starlight-go/starlight"
+	"reflect"
+)
+
+type Runtime int64
+
+const (
+	Golang Runtime = iota
+	Javascript
+	Starlark
+)
+
+func CallFunc(name string, r Runtime, code string, arg interface{}) (interface{}, error) {
```

Comment:
I'm noticing the awkwardness of "go" vs the "interpreted" languages -- one needing the name (or some kind of handle) and one needing the actual code text. I wonder if we could basically box that difference behind an interface -- maybe something like FunctionHandle.

```
type FunctionHandle interface {
Invoke(arg) arg
}

type GoFunctionHandle struct {
name string
}

type StarlarkFunctionHandle struct {
code string
}
```

Reply:
But I can't tell if that's a bit overdesigned for where we are now.

Reply:
In the medium term, I'd like to see modules be able to provide their own "runtimes". But I'm cool with punting on that until we have more uses of them built out.

Reply:
Are you thinking about having the CallFunc get passed in a FunctionHandle from the db?

Reply:
That was the thought, but TBH I'm feeling like maybe we should just punt on it for now and merge this part in as is!

Reply:
Actually I got it. Just cleaned this up.